src/Core/src/Handlers/SearchBar/SearchBarHandler.Android.cs

```diff
@@ -135,7 +135,10 @@ public static void MapFocus(ISearchBarHandler handler, ISearchBar searchBar, obj
 		void OnQueryTextSubmit(object? sender, QueryTextSubmitEventArgs e)
 		{
 			VirtualView.SearchButtonPressed();
```

Comment:
Here, we must clear the focus too:

`(sender as SearchView)?.ClearFocus();`

Similar to https://github.com/xamarin/Xamarin.Forms/blob/2f8f4864a4d289dc89a6228e2ca9d6a49993e365/Xamarin.Forms.Platform.Android/Renderers/SearchBarRenderer.cs#L49

Reply:
> Here, we must clear the focus too:
> 
> `(sender as SearchView)?.ClearFocus();`
> 
> Similar to https://github.com/xamarin/Xamarin.Forms/blob/2f8f4864a4d289dc89a6228e2ca9d6a49993e365/Xamarin.Forms.Platform.Android/Renderers/SearchBarRenderer.cs#L49

Hi @jsuarezruiz 
We have replaced the Virtual view unfocus method with **(sender as UISearchBar)?.ResignFirstResponder();** . Could you please verify these changes?